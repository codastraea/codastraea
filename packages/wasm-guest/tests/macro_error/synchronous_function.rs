use serpent_automation_wasm_guest::workflow;

#[workflow]
fn counter() {}

fn main() {}
