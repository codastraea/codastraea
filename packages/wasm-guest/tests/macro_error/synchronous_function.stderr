error: `workflow` functions should be async
 --> tests/macro_error/synchronous_function.rs:4:1
  |
4 | fn counter() {}
  | ^^^^^^^^^^^^
